dev/src/write-batch.ts

```diff
@@ -103,7 +103,7 @@ interface WriteOp {
 export class WriteBatch {
   private readonly _firestore: Firestore;
   private readonly _serializer: Serializer;
-  private readonly _writes: WriteOp[] = [];
+  private readonly _ops: Array<() => WriteOp> = [];
```

Comment:
This probably deserves a comment...  Part of me wants to rename it lazyOps too, though not sure that is necessary /better.

Reply:
Comment added.